azure-pipelines-dev.yml

```diff
@@ -195,22 +195,10 @@ stages:
             displayName: 'Update pip'
 
           - script: |
-                pip install numpy
-                pip install pandas
-                pip install scipy
-            displayName: 'Install numpy, pandas, and scipy'
-
-          - script: |
-              pip install --requirement requirements.txt
-              echo "about to run pip install --requirement reqs/requirements-dev-test.txt --constraint constraints-dev.txt"
-              pip install --requirement reqs/requirements-dev-test.txt --constraint constraints-dev.txt
-              pip install  .
+              pip install --constraint constraints-dev.txt ".[test]" pytest-azurepipelines
```

Comment:
`numpy` `pandas` etc are all required for GX. This change ensures that we just do the installation once.